src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -145,6 +145,58 @@ bool Lowering::IsContainableImmed(GenTree* parentNode, GenTree* childNode) const
     return false;
 }
 
+#ifdef TARGET_ARM64
+//------------------------------------------------------------------------
+// IsContainableBinaryOp: Is the child node containable from the parent node?
```

Comment:
Need to update this comment to: "Is the child node a binary op that is containable from the parent node?'